apps/dc_tools/odc/apps/dc_tools/utils.py

```diff
@@ -177,6 +184,9 @@ def index_update_dataset(
                         )
                 else:
                     logging.warning("Dataset already exists, not indexing")
```

Comment:
We know the `id` of the dataset we're processing here, it'd be worth including in the log output.